crossing-repair.lic

```diff
@@ -12,7 +12,7 @@ class CrossingRepair
   def initialize
     setup
 
-    ensure_copper_on_hand(@repair_withdrawal_amount, @bank_info)
```

Comment:
Sounds good to me.

:+1: for consistency in a future PR. Let me know how I can help.